src/bot.ts

```diff
@@ -204,6 +204,13 @@ client.on('ready', async () => {
             await timeoutManager.registerTimeout(TimeoutType.WeeklyXpUpdate, getNextFridayEvening(), { pastStrategy: PastTimeoutStrategy.Invoke });
         }
 
+        // Register global slash commands on startup
+        if (client.application) {
```

Comment:
This can run without me having to define `guildId` and `clientId` in the auth file, right?

Reply:
Correct, it doesn't need the guildId in the first place and the clientId is known.